src/components/formRow/formRow.comp.tsx

```diff
@@ -190,6 +190,8 @@ export const FormRow = withAppContext(({ context, field, direction, showReset, o
         return <input type="file" accept={field.accept || '*'} placeholder={field.placeholder || 'Select file...'} name={field.name || 'file'} disabled={field.readonly} required={field.required} />;
       case 'note':
         return <p className="note">{field.value}</p>;
+      case 'date':
+        return <input type="date" {...inputProps('Enter date...')}/>;
```

Comment:
Does all date inputs have the same format? or is it something you can specify?

Reply:
ISO format _"2018-12-31"_ there is a format parameter but I don't think it fits the actual design of RESTool

Reply:
👍🏻